<launch>

  <!-- WALLFOLLOWING -->
    <node name="wallfollowing" pkg="hanse_wallfollowing" type="wallfollowing.py" output="screen" />
    <node name="echosounderwalldetection" pkg="hanse_wallfollowing" type="echosounderwalldetection" output="screen" />


  <!-- ANGULAR PID -->

  <group ns="wallfollowing_angular_pid">    
    <node name="pid" pkg="hanse_pidcontrol" type="pid_control" output="screen">
      <rosparam file="$(find hanse_wallfollowing)/config_angular_pid.yaml"/>
    </node>
  </group>
</launch>
